test/metabase/lib/js_test.cljs

```diff
@@ -338,9 +339,10 @@
     (testing "segment refs come without options"
       (is (= [["segment" segment-id]]
              (->> query lib/available-segments (map to-legacy-refs)))))
-    (testing "metric refs come without options"
-      (is (= [["metric" metric-id]]
-             (->> metric-query lib/available-metrics (map to-legacy-refs)))))
+    ;; TODO FIXME
```

Comment:
@metamben

It seems these started failing after some metrics changes in 8d22b08feeac5127b08851a02c544ee326645628

I've commented them out for now so they show up in the diff, but I will either fix or remove them. Let me know which case you think applies:

1. The tests are no longer relevant and can be removed.
2. The tests are still relevant, but need to be updated to work.
3. The tests are fine as-is and indicate a bug that should be fixed.

Reply:
Failing test is resolved now thanks to @metamben